edge/pkg/metamanager/client/configmap.go

```diff
@@ -51,7 +51,7 @@ func (c *configMaps) Delete(name string) error {
 }
 
 func (c *configMaps) Get(name string) (*api.ConfigMap, error) {
-	resource := fmt.Sprintf("%s/%s/%s", c.namespace, model.ResourceTypeConfigmap, name)
```

Comment:
Has anything wrong with configmap's ns?

Reply:
Test `default` and `test` namespace, both ok.